ui/raidboss/data/05-shb/trial/memoria_misery_ex.txt

```diff
@@ -0,0 +1,96 @@
+# 
+hideall "--sync--"
+
+# Phase 1
+0 "Start!"
+0.0 "--sync--" sync /:Engage!/ window 0,1
+0.9 "--sync--" sync /:Varis Yae Galvus:4CEF:/ window 1,0
+19.0 "Altius" sync /:Varis Yae Galvus:4CCA:/
+28.9 "Alea Iacta Est" sync /:Varis Yae Galvus:4CD2:/
+34.1 "Terminus Est" # sync /:Terminus Est:4CB4:/
+45.1 "Citius" sync /:Varis Yae Galvus:4CF0:/
+56.0 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+64.2 "Ignis Est Action"
+64.2 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+82.4 "Electrified Gunshield" sync /:Varis Yae Galvus:4CD7:/
+91.5 "Altius" sync /:Varis Yae Galvus:4CCA:/
+100.4 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+101.6 "Terminus Est" # sync /:Terminus Est:4CB4:/
+108.5 "Ignis Est Action" # sync /:Ignis Est:4CB6:/
+109.6 "Magitek Shock" sync /:Varis Yae Galvus:4CDA:/
+120.7 "Reinforced Gunshield" sync /:Varis Yae Galvus:4CD9:/
+128.9 "Citius" sync /:Varis Yae Galvus:4CF0:/
+132.0 "Magitek Shielding" sync /:Varis Yae Galvus:4CDB:/
+147.2 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/
+156.0 "Ventus Est Ready" sync /:Varis Yae Galvus:4CC6:/
+163.6 "Magitek Burst" sync /:Varis Yae Galvus:4CDE:/
+164.2 "Ventus Est Action"
+166.7 "Magitek Burst Hit"
+177.1 "Vivere Militare Est" sync /:Varis Yae Galvus:4CCC:/
+
+# Phase 2
+200.1 "--targetable--"
+233.2 "Vivere Militare Est" sync /:Bladesblood:4CCF:/ duration 30 window 80,50
+
+# Phase 3 (random groups)
+262.3 "Gunshield Actions"
+262.3 "--sync--" sync /:Varis Yae Galvus:4CD8:/ window 80,20 jump 1262.3
+262.3 "--sync--" sync /:Varis Yae Galvus:4CD9:/ window 80,20 jump 2314.1
+262.3 "--sync--" sync /:Varis Yae Galvus:4CD7:/ window 80,20 jump 3368.8
+268.0 "Citius" sync /:Varis Yae Galvus:4CF0:/ window 60,20 jump 244.0
```

Comment:
That is awesome, I have not thought out this method!
But since Varis would use `Citius` after the first and second group, and would not use it at the end of phase 3, so that is exactly what I get troubled. I will follow what you said to fix it and rename as `varis-ex.txt`.

Reply:
I can't take credit for it. quisquous was kind enough to point the method out to me when I submitted the Ala Mhigo timeline.